sdk/servicebus/service-bus/src/serviceBusAtomManagementClient.ts

```diff
@@ -0,0 +1,547 @@
+   *                                                                                 must attempt to update the resource or to create new one.
+   * @param {ResourceSerializer} serializer                                          The XML serializer to use.
+   * @return {Promise<HttpOperationResponse>}
+   */
+  private async _createResource(
+    path: string,
+    requestBody: any,
+    isUpdate: boolean = false,
+    serializer: ResourceSerializer
+  ): Promise<HttpOperationResponse> {
+    let webResource: WebResource = new WebResource();
+
+    webResource.method = "PUT";
+    webResource.url = this.getUrl(path);
+    webResource.body = JSON.stringify(requestBody);
+    if (isUpdate) {
+      webResource.headers.set("If-Match", "*");
+    }
+    webResource.headers.set("content-type", "application/atom+xml;type=entry;charset=utf-8");
+    webResource.headers.set("content-length", Buffer.byteLength(webResource.body));
+
+    const atomXmlOperationSpec: AtomXmlOperationSpec = {
+      serializer: serializer,
+      shouldParseResponse: false
+    };
+    webResource.atomXmlOperationSpec = atomXmlOperationSpec;
+
+    return this.sendRequest(webResource);
+  }
+
+  /**
+   * Gets a resource.
+   *
+   * @param {string}             path                                                The resource path.
+   * @param {ResourceSerializer} serializer                                          The XML serializer to use.
+   * @return {Promise<HttpOperationResponse>}
+   */
+  private async _getResource(
+    path: string,
+    serializer: ResourceSerializer
+  ): Promise<HttpOperationResponse> {
+    let webResource: WebResource = new WebResource();
+
+    webResource.method = "GET";
+    webResource.url = this.getUrl(path);
+
+    const atomXmlOperationSpec: AtomXmlOperationSpec = {
+      serializer: serializer,
+      shouldParseResponse: true
+    };
+    webResource.atomXmlOperationSpec = atomXmlOperationSpec;
+
+    return this.sendRequest(webResource);
+  }
+
+  /**
+   * Lists resources.
+   *
+   * @param {string}             path                                                The resource path.
+   * @param {object}             requestBody                                         The resource handler.
+   * @param {ResourceSerializer} serializer                                          The XML serializer to use.
+   * @return {Promise<HttpOperationResponse>}
+   */
+  private async _listResources(
+    path: string,
+    requestBody: any,
+    serializer: ResourceSerializer
+  ): Promise<HttpOperationResponse> {
+    let webResource: WebResource = new WebResource();
+
+    webResource.method = "GET";
+
+    const queryParams: any = {};
+    if (requestBody) {
+      if (requestBody.skip) {
+        queryParams["$skip"] = requestBody.skip;
+      }
+      if (requestBody.top) {
+        queryParams["$top"] = requestBody.top;
+      }
+    }
+
+    webResource.url = this.getUrl(path, queryParams);
+
+    console.log(webResource.url);
```

Comment:
Make sure we don't have console.log statements before merging. If we want to log something, we should use the loggers so users can configure when to enable them.